### DISCUSSION TITLE
LISTA EPG de Portugal

### DISCUSSION THREAD

**Discussion by Vicanpor** [LITUATUI/M3UPT#54] [General]
Hola Buenas tardes 
Hay alguna lista que conzcais de EPG para los canales Portugueses , utilizo el OTT Navigator.
Mil Gracias

**Comment by Vicanpor** [comment]
las de LITUATUI no me funcionan.

**Comment by LITUATUI** [comment]
Use the [backup code](https://m3upt.com/docs/#-ott-navigator-iptv) for the OTT Navigator.
